src/lib/util/rand.c

```diff
@@ -34,6 +34,8 @@ static _Thread_local bool fr_rand_initialized = false;
 void fr_rand_init(void)
 {
 	int fd;
+	uint8_t *p = (uint8_t *) &fr_rand_pool.randrsl[0];
+	uint8_t *end = p + sizeof(&fr_rand_pool.randrsl);
```

Comment:
Shouldn't this be `sizeof(fr_rand_pool.randrsl)`?

Reply:
Will double check; thanks.

Reply:
Indeed it should; correcting. Thanks.